plane_mcp/auth/plane_oauth_provider.py

```
@@ -0,0 +1,360 @@
+                installations_response = await client.get(
+                    f"{base_url}/auth/o/app-installation/",
+                    headers={
+                        "Authorization": f"Bearer {token}",
+                        "Content-Type": "application/json",
+                    },
+                )
+
+                installations: list[PlaneOAuthAppInstallation] = installations_response.json()
+
+                if not installations:
+                    raise ValueError("No app installations found")
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)